.github/workflows/ci.yml

```diff
@@ -26,13 +26,17 @@ jobs:
           version: 9.5.0
           run_install: false
 
-      - name: Setup UV
-        run: |
-          python3 -m pip install --upgrade pip
-          pip install uv
+      - name: Install Python, UV and dependencies
+        uses: astral-sh/setup-uv@v5
+        with:
+          python-version: "3.11"
+          version: "0.6.14"
 
       - name: Run moon ci
         run: moon ci
+        env:
+          PUBLIC_LANGGRAPH_API_URL: 'https://dummy-langgraph-url.com'
```

Comment:
Locally, I have `PUBLIC_LANGGRAPH_API_URL=http://127.0.0.1:2024`

AFAIK this will be a requirement in production.